source/fundamentals/crud/read-operations/project.txt

```diff
@@ -71,11 +71,10 @@ field of each document:
    }
 
 The projection document specifies a value of ``1`` for ``name`` to
-indicate that the read operation result should *include* the ``name``
-field of each returned document. As a result, this projection implicitly
-excludes the ``qty`` and ``rating`` fields. Passing this projection to
-``find()`` with an empty query document and no sort document yields
-the following results:
+instruct the operation to *include* the ``name`` field of each returned document in
+the results. As a result, this projection implicitly excludes the ``qty`` and
+``rating`` fields. Passing this projection to ``find()`` with an empty query document
+and no sort document yields the following results:
```

Comment:
S: sounds a little funny to say "result" twice so close together with different meanings

you could end the first sentence after ``name``, then pick up here:

```suggestion
This instructs the operation to *include* the ``name`` field of each returned document in
the results and *exclude* the ``qty`` and
``rating`` fields. Passing this projection to ``find()`` with an empty query document
and no sort document yields the following results:
```